sdk/test-utils/recorder/package.json

```diff
@@ -59,6 +59,7 @@
     "@azure/core-http": "^1.0.0",
     "@opentelemetry/types": "^0.2.0",
     "fs-extra": "^8.1.0",
+    "match-all":"^1.2.5",
```

Comment:
looks like this will be a built-in eventually: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/matchAll